bundles/org.openhab.binding.dwdpollenflug/src/main/resources/ESH-INF/thing/region.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dwdpollenflug"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="region">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		
+		<label>DWD Pollen count index - Region</label>
+		<description>Pollen count index for a region or partregion</description>
+
+		<channel-groups>
+			<channel-group id="alder" typeId="pollentype">
+				<label>Alder</label>

```

Comment:
I think the labels shpould be localized, too.

Reply:
I think I already do this in i18n file?

Reply:
My fault.